app/routes/__index/dns-records/instructions.tsx

```diff
@@ -27,55 +31,107 @@ export default function CertificateInstructionsRoute() {
-                for your domain.
-              </Text>
-            </ListItem>
-          </UnorderedList>
+          <Tabs>
+            <TabList>
+              <Tab>
+                <Text fontWeight="medium">A Record</Text>
+              </Tab>
+              <Tab>
+                <Text fontWeight="medium">AAAA Record</Text>
+              </Tab>
+              <Tab>
+                <Text fontWeight="medium">CNAME Record</Text>
+              </Tab>
+              <Tab>
+                <Text fontWeight="medium">TXT Record</Text>
+              </Tab>
+            </TabList>
+
+            <TabPanels>
+              <TabPanel>
+                <Text>
+                  <Text as="span" fontWeight="bold">
+                    Purpose:{' '}
+                  </Text>
+                  Maps a domain name to an IPv4 address.
+                </Text>
+                <Text>
+                  <Text as="span" fontWeight="bold">
+                    Example:{' '}
+                  </Text>
+                  Your project's domain name is "my-project.user.mystudentproject.ca" and its IPv4
+                  address is "192.168.0.1". Create an A record to associate these two.
+                </Text>
+              </TabPanel>
+              <TabPanel>
+                <Text>
+                  <Text as="span" fontWeight="bold">
+                    Purpose:{' '}
+                  </Text>
+                  Maps a domain name to an IPv6 address.
+                </Text>
+                <Text>
+                  <Text as="span" fontWeight="bold">
+                    Example:{' '}
+                  </Text>
+                  Your project's domain name is "my-ipv6-project.user.mystudentproject.ca" and its
+                  IPv6 address is "2001:0db8:85a3:0000:0000:8a2e:0370:7334". Create an AAAA record
+                  to associate these two.
+                </Text>
+              </TabPanel>
+              <TabPanel>
+                <Text>
+                  <Text as="span" fontWeight="bold">
+                    Purpose:{' '}
+                  </Text>
+                  Maps a domain name to another domain name.
+                </Text>
+                <Text>
+                  <Text as="span" fontWeight="bold">
+                    Example:{' '}
+                  </Text>
+                  Your project's domain name is "my-project.user.mystudentproject.ca", and you also
+                  have a custom domain called "my-custom-domain.com". Create a CNAME record to map
+                  "my-custom-domain.com" to "my-project.user.mystudentproject.ca".
+                </Text>
+                <Text>
+                  <Text as="span" fontWeight="bold">
+                    Note:{' '}
+                  </Text>
+                  When creating a CNAME record, remember to only include the domain name in the
+                  alias field. Including other parts of a URL, such as "https://" or a trailing "/",
```

Comment:
Switch "alias" to "value," since we don't have anything called "alias" in our form.